### ISSUE TITLE
Missing Incorrect heading order

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#1235]
On https://roadwarriorprd.wpengine.com/b-corp/ EDAC isn't flagging incorrect heading order when it should. @pattonwebz can you please check to make sure this is not a bug?

**Comment by SteveJonesDev** [comment]
Fixed in PR: equalizedigital/accessibility-checker#1234<br>Released in version 1.32.0: [https://github.com/equalizedigital/accessibility-checker/releases/tag/v1.32.0](https://github.com/equalizedigital/accessibility-checker/releases/tag/v1.32.0)
